src/main/kotlin/commands/IssueCommand.kt

```diff
@@ -0,0 +1,128 @@
+package commands
+
+import AuthConfig
+import Command
+import ConfigType
+import FileManager
+import Main
+import arg
+import org.kamiblue.api.issue.Issue
+import org.kamiblue.api.pull.PullRequest
+import com.google.gson.Gson
+import doesLater
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import string
+
+/**
+ * @author sourTaste000
+ * @since 2020/9/8
+ */
+@Suppress("BlockingMethodInNonBlockingContext")
+object IssueCommand : Command("issue") {
+    init {
+        /*
+        TODO:
+             Make a tree that defaults to kami-blue for user
+             Support Multiple Assignees
+         */
+        string("user") {
+            string("repoName") {
+                string("issueNum") {
+                    doesLater { context ->
+                        val user: String = context arg "user"
+                        val repoName: String = context arg "repoName"
+                        val issueNum: String = context arg "issueNum"
+                        val githubToken =
+                            FileManager.readConfigSafe<AuthConfig>(ConfigType.AUTH, true)?.githubToken ?: run {
+                                message.channel.send {
+                                    embed {
+                                        title = "[$name]"
+                                        description = "Github Token not found!"
+                                        color = Main.Colors.ERROR.color
+                                    }
+                                }
+                                return@doesLater
+                            }
+                        val request = Request.Builder().addHeader("Authorization", "token $githubToken").url("https://api.github.com/repos/$user/$repoName/issues/$issueNum").get().build()
```

Comment:
![image](https://user-images.githubusercontent.com/17222512/93225791-9cde3f80-f740-11ea-9484-559b5e4d88d1.png)
This format doesn't look right

Reply:
i tried to fetch /rate_limit with that, works correctly